inc/utils.php

```diff
@@ -81,3 +81,25 @@ function cardToken()
 {
     return $cardToken = filter_input(INPUT_POST, 'cardToken', FILTER_SANITIZE_STRING) ?: '';
 }
+
+/**
+ * Isolates static debug calls.
+ *
+ * @param  string $message
+ */
+function debug($message)
+{
+    Mollie_WC_Plugin::debug($message);
+}
```

Comment:
@mmaymo Sorry we missed to pass the second parameter in this function, could you check it please?